src/main/java/com/minecolonies/coremod/client/gui/WindowHutBarracksTower.java

```diff
@@ -0,0 +1,26 @@
+package com.minecolonies.coremod.client.gui;
+
+import com.minecolonies.coremod.colony.buildings.AbstractBuildingGuards;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Our building hut view
+ *
+ * @author Vroni
+ */
+public class WindowHutBarracksTower extends WindowHutGuardTower {
+    /**
+     * Constructor for the window of the worker building.
+     *
+     * @param building class extending {@link AbstractBuildingGuards.View}.
+     */
+    public WindowHutBarracksTower(AbstractBuildingGuards.View building) {
+        super(building);
+    }
+
+    @NotNull
+    @Override
+    public String getBuildingName() {
+        return "com.minecolonies.coremod.gui.workerhuts.BarracksTower";
```

Comment:
This file needs propper formatting, the lang string has to be fully lower case too. @author is not needed.